esp/esp/themes/theme_data/fruitsalad/less/main.less

```diff
@@ -329,8 +329,10 @@ text-transform: uppercase;
 
 #menu #tab_logo #logo {
 border: none;
-margin-left: 9px;
-margin-top: 9px;
+padding-top: 9px;
+margin-left: auto;
+margin-right: auto;
+display: block;
```

Comment:
Why did you make these changes to the logo styling?

Reply:
oops that must be residual from working on the logo centering PR at a similar time. I'll fix that!